packages/maskbook/src/plugins/PoolTogether/UI/PoolView.tsx

```diff
@@ -277,7 +277,7 @@ export function PoolView(props: PoolProps) {
                                     classes={{ icon: classes.poolIcon }}
                                 />
                                 {t('plugin_pooltogether_apr', {
-                                    apr: pool.tokenFaucets[0].apr.toFixed(2),
+                                    apr: pool.tokenFaucets[0].apr?.toFixed(2) ?? 0,
```

Comment:
What if tokenFaucets[0] is undefined?

Reply:
There is a check of `pool.tokenFaucets.length !== 0` above, and we can assume that members of`pool.tokenFaucets` aren't nullable.

But, checking the first member of `tokenFaucets` would make the logic more reliable.

Reply:
Just updated the code.